R/annotation.R

```diff
@@ -91,12 +91,31 @@ add_name_lists_anno <- function(eset, name_lists, genes = T) {
 #'@param genes if TRUE apply to rows, otherwise columns
 #'@return  annotated Expression set
 add_data_stats_anno <- function(eset, genes = T) {
+  stat_fns <- list(
```

Comment:
You should add @import statement similar to other methods 
![image](https://github.com/user-attachments/assets/f188aa17-c445-42af-8195-2a64ce324829)